### DISCUSSION TITLE
Templates

### DISCUSSION THREAD

**Discussion by josejachuf** [salvo-rs/salvo#94] [General]
Hi,

Salvo works with some template system?

Jose

**Comment by chrislearn** [comment]
You can use any template engine you like.

**Comment by jhjacobs81** [comment]
I have found Askama the most easy to work with :) but im very much a beginner :)

**Reply by chrislearn** [reply]
Template engine is just generate string,
I have added a new example: https://github.com/salvo-rs/salvo/tree/main/examples/template-askama
You can run it with this command:

```
cargo run --bin example-template-askama
```

Then use browser to view page: http://localhost:7878/jhjacobs81

**Reply by jhjacobs81** [reply]
Thank you :) I have tried a few web frameworks like actix, rocket, rouille and Warp, but i havent gotten as far with any of them as i have with Salvo :) It's very easy to read and understand!
